guides/handling_client_actions.md

```diff
@@ -0,0 +1,73 @@
+## Handling Client Actions
```

Comment:
remove this doc in lieu of the actions and events doc I am assembling

Reply:
Removed as suggested. Thanks!